neuralforecast/core.py

```diff
@@ -463,6 +585,9 @@ def predict_rolled(
             DataFrame with insample `models` columns for point predictions and probabilistic
             predictions for all fitted `models`.
         """
+        print(
+            "WARNING: this method will be deprecated. Use `cross_validation` or `predict_insample` instead."
+        )
```

Comment:
I prefer using `warning.warn` instead of `print` but it is just a preference. Not blocking.